### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Thinus01**
Created comment popup window.
Popup window contains:
   descriptive information about the product
   Image about product
   shows amounts of comments
   shows comments posted
   shows an area to add a comment

### CODE REVIEW

File: src/modules/commentPage.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const commentPage = require('./commentPage.js');
+
```
Comment (cherelemma) [inline src/modules/commentPage.test.js:2]: The test is not working. I suggest you try it by creating a separate function for the comment counter. Moreover, please add more cases to test the edge cases.

File: src/modules/commentPage.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <p id='Close'>X</p>
+        <image class="imgContainer" src = "${objectDetails.primaryImage}" alt="Object Image"/>
+        <h2 class='title1 detail'>${objectDetails.title}</h2>
+        <p class='detail1'>Artist Display Name: ${objectDetails.artistDisplayName}</p>
+        <p class='detail2'>Dimensions: ${objectDetails.dimensions}</p>
+        <p class='detail1'>Object name: ${objectDetails.objectName}</p>
+        <p class='detail2'>Object Date: ${objectDetails.objectDate}</p>
+        <p class='detail1'>City: ${objectDetails.city}</p>
+        <p class='detail2'>Country: ${objectDetails.country}</p>
+        <p id='dComment'>Comments (${comments.length})</p>`;
```
Comment (cherelemma) [inline src/modules/commentPage.js:20]: For the test case to work, it is better if you create a separate function for the comment counter.

**Review by cherelemma — COMMENTED** [review]
Hello @Thinus01 👍 ,

Great job so far!
There are some issues that you still need to work on to meet the required requirements.
image

# Highlights 
Linters check failed ❌ 
Git flow observed ✔️

# Required Changes ♻️

**Review by cherelemma — APPROVED** [review]
# Status: Approved 👍
